examples/platforms/simulation/radio.c

```diff
@@ -1016,6 +1055,55 @@ otError otPlatRadioGetCoexMetrics(otInstance *aInstance, otRadioCoexMetrics *aCo
 }
 #endif
 
+uint64_t otPlatRadioGetNow(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    return otPlatTimeGet();
+}
+
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+static otError updateIeData(otInstance *aInstance)
```

Comment:
Is it necessary to define this static method?
 It seems to me that this simple method is only used  once in `otPlatRadioEnableCsl()`.

Reply:
It would soon be used in linke metrics. I think it would be a waste if I expand it here and then reencapsulate it in link metrics.

Reply:
If so, fine to me to keep it.